pipeline.ipynb

```diff
@@ -146,13 +175,13 @@
     "    'thres_corr': 0.9}\n",
     "param_second_spatial = {\n",
     "    'dl_wnd': 5,\n",
-    "    'sparse_penal': 0.05,\n",
+    "    'sparse_penal': 0.1,\n",
     "    'update_background': False,\n",
     "    'post_scal': True,\n",
     "    'zero_thres': 'eps'}\n",
     "param_second_temporal = {\n",
     "    'noise_freq': 0.06,\n",
-    "    'sparse_penal': 1,\n",
+    "    'sparse_penal': 20,\n",
```

Comment:
not sure that's a good choice for most of the data out there. If this is specific to certain data let's not changing the parameters back-and-forth

Reply:
This was changed back to `sparse_penal : 1` in commit`a6727ba`